ado_asana_sync/sync/app.py

```diff
@@ -45,10 +52,10 @@ class App:
 
     def __init__(
         self,
-        ado_pat: str = None,
-        ado_url: str = None,
-        asana_token: str = None,
-        asana_workspace_name: str = None,
+        ado_pat: str = "",
+        ado_url: str = "",
+        asana_token: str = "",
+        asana_workspace_name: str = "",
     ) -> None:
```

Comment:
**Suggestion:** Avoid using mutable default arguments in the function definition.
```suggestion
    def __init__(
        self,
        ado_pat: str = None,
        ado_url: str = None,
        asana_token: str = None,
        asana_workspace_name: str = None,
    ) -> None:
        self.ado_pat = ado_pat if ado_pat is not None else os.environ.get("ADO_PAT")
        self.ado_url = ado_url if ado_url is not None else os.environ.get("ADO_URL")
```